doc/tutorial/basic/tutorial.rst

```diff
@@ -344,7 +344,7 @@ once will overwrite what was learned by any previous ``fit()``::
     max_iter=-1, probability=False, random_state=None, shrinking=True,
     tol=0.001, verbose=False)
   >>> clf.predict(X_test)
-  array([1, 0, 1, 1, 0])
+  array([0, 0, 0, 1, 0])
```

Comment:
these samples are generated randomly (maybe we should avoid doing so in the tutorial?)